createuserpkg

```diff
@@ -36,16 +35,20 @@ def main():
     required_user_options = optparse.OptionGroup(
         parser, 'Required User Options')
     required_user_options.add_option(
-        '--name', '-n', help='User shortname. REQUIRED.')
+        '--name', '-n',
+        help='User shortname. REQUIRED.')
     required_user_options.add_option(
```

Comment:
You've made a bunch of changes unrelated to your core changes. This makes reviewing your changes harder.

Reply:
sorry about that, thought it would be easier to read